root/static/js/canto-modules.js

```diff
@@ -6374,6 +6374,11 @@ var annotationEditDialogCtrl =
         if (organisms.length == 1) {
```

Comment:
Perhaps the check to see if there is only one organism should go after the filter for host or pathogen?

Reply:
Yes, that makes more sense. I'll push a change for that.